pkg/distributor/writefailures/cfg.go

```diff
@@ -0,0 +1,18 @@
+package writefailures
+
+import (
+	"flag"
+	"path"
+)
+
+type Cfg struct {
+	AddInsightsLabel bool `yaml:"add_insights_label"`
+	LogRate          int  `yaml:"logging_rate"`
+}
+
+// RegisterFlags registers distributor-related flags.
+func (cfg *Cfg) RegisterFlagsWithPrefix(prefix string, fs *flag.FlagSet) {
+	fs.BoolVar(&cfg.AddInsightsLabel, path.Join(prefix, "add-insights-label"), false, "Whether a insight=true should be added to all log messages or not.")
```

Comment:
Can you expand on why we need this? Seems like we're bleeding our Grafana Cloud requirement into OSS. Can't we just always include the `insights=true` label?

Reply:
It wasn't exactly necessary but I made it configurable in case some user prefer to not have it. Anyway, since this is just in the beginning and this config is experimental I decided to remove it for now. Thank you.